examples/tests/valid/std_containers.w

```diff
@@ -40,4 +40,13 @@ let immutMap = mutMap.copy();
 assert(sMap.get("one") == 1);
 assert(sMap.size() == 2);
 assert(immutMap.size() == 3);
-assert(nestedMap.get("a").get("b").get("c") == "hello");
\ No newline at end of file
+assert(nestedMap.get("a").get("b").get("c") == "hello");
+
+class Animal {}
+class Cat extends Animal {}
+class Dog extends Animal {}
+
+let heterogeneousArray = Array<Animal>[
+  new Cat(),
+  new Dog(),
+];
```

Comment:
Can we battle-test this a little bit more? For example, adding `Set<Animal>`, `Map<Animal>`, or `Array<Array<Animal>>` test cases?